assets/css/pricing-modal.css

```diff
@@ -1338,3 +1360,141 @@ p.imagify-modal-sidebar-title.imagify-modal-sidebar-title {
 		left: 10px;
 	}
 }
+
+/** Ribbon **/
+.ribbon {
+	position: absolute;
+	left: -5px; top: -5px;
+	z-index: 1;
+	overflow: hidden;
+	width: 75px; height: 75px;
+	text-align: right;
+}
+.ribbon span {
+	font-size: 0.9em;
+	font-weight: bold;
+	color: #FFF;
+	text-align: center;
+	line-height: 22px;
+	transform: rotate(-45deg);
+	-webkit-transform: rotate(-45deg);
+	width: 100px;
+	display: block;
+	background: #79A70A;
+	background: linear-gradient(#3FB1CF 0%, #3FB1CF 100%);
+	box-shadow: 0 3px 10px -5px rgba(0, 0, 0, 1);
+	position: absolute;
+	top: 19px; left: -21px;
+}
+.ribbon span::before {
+	content: "";
+	position: absolute; left: 0px; top: 100%;
+	z-index: -1;
+	border-left: 3px solid #3FB1CF;
+	border-right: 3px solid transparent;
+	border-bottom: 3px solid transparent;
+	border-top: 3px solid #3FB1CF;
+}
+.ribbon span::after {
+	content: "";
+	position: absolute; right: 0px; top: 100%;
+	z-index: -1;
+	border-left: 3px solid transparent;
+	border-right: 3px solid #3FB1CF;
+	border-bottom: 3px solid transparent;
+	border-top: 3px solid #3FB1CF;
+}
+/** End Ribbon **/
+
+/** Toggle Switch **/
+.toggle-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 10px;
+    position: relative;
+    width: 70%;
+    margin: 80px auto 40px;
+}
+.toggle-container .toggle-label {
+    font-size: 16px;
+    font-weight: 500;
+}
+.toggle-container .toggle-label:nth-of-type(2) {
+    color: #c8ced5;
+}
+.switch {
+    position: relative;
+    display: inline-block;
+    width: 48px;
+    height: 24px;
+}
+.switch input {
+    opacity: 0;
+    width: 0;
+    height: 0;
+}
+.slider {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: #97c45e;
+    border-radius: 30px;
+    cursor: pointer;
+    transition: 0.4s;
+}
+.slider:before {
+    position: absolute;
+    content: "";
+    height: 18px;
+    width: 18px;
+    left: 4px;
+    bottom: 3px;
+    background-color: white;
+    border-radius: 50%;
+    transition: 0.4s;
+}
+input:checked + .slider {
+    background-color: #97c45e;
+}
+input:checked + .slider:before {
+    transform: translateX(22px);
+}
+.badge-container {
+    align-items: center;
+    gap: 5px;
+	position:absolute;
+	right: 80px;
+	top: -25px;
+}
+.badge {
+    font-size: 12px;
+    padding: 4px 10px;
+    border: 1px solid #b8bfc5;
+    border-radius: 3px;
+    color: #b8bfc5;
+	font-weight: 700;
+    transition: background-color 0.4s, color 0.4s;
+}
+.badge.badge-checked {
+	background-color: #E2F0D1;
+	color: #8bc34a;
+	border: 1px solid #8bc34a
+}
+.arrow-container {
+	position: absolute;
+    right: 180px;
+    top: 0px;
+}
+.toggle-container img:nth-of-type(2) {
+    display: none;
+}
+input:checked ~ .badge-container .badge {
+    background-color: #97c45e;
+    color: white;
+    border-color: #97c45e;
+    opacity: 1; 
+}
+/** End Toggle Switch **/
```

Comment:
I would suggest to prefix all those classes with `imagify-` to make sure it doesn't conflict with anything else, the names are too generic currently